specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimapis.json

```diff
@@ -1801,6 +1801,654 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiIdRevParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "|     Field     |     Usage     |     Supported operators     |     Supported functions     |</br>|-------------|-------------|-------------|-------------|</br>| name | filter | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |</br>| displayName | filter | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |</br>| description | filter | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |</br>| path | filter | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |</br>"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/TopQueryParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SkipQueryParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A collection of resolvers summary entities at the GraphQL API level.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ResolverCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the resolver failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "./definitions.json#/definitions/ResolverContract"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/resolvers/{resolverId}": {
+      "head": {
+        "tags": [
+          "GraphQLApiResolver"
+        ],
+        "operationId": "GraphQLApiResolver_GetEntityTag",
+        "description": "Gets the entity state (Etag) version of the GraphQL API resolver specified by its identifier.",
+        "x-ms-examples": {
+          "ApiManagementHeadGraphQLApiResolver": {
+            "$ref": "./examples/ApiManagementHeadGraphQLApiResolver.json"
+          }
+        },
+        "parameters": [
```

Comment:
>parameters [](http://example.com/codeflow?start=9&length=10)

There are a bunch of breaking changes reported on this PR. Please get a breakingchanges approval and ARM can review the changes afterwards. Once you have the approval please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers

Reply:
@hoonality I see the target branch here is [Azure:release-apimangement-2022-08-01](https://github.com/Azure/azure-rest-api-specs/tree/release-apimangement-2022-08-01) - is that a development branch, and this is a new API in development that is not deployed yet?